### ISSUE TITLE
will this be available for arm devices?

### ISSUE THREAD

**Issue by rubenix** [Monitorr/Monitorr#229]
would love to have it running on my pi; can I help somehow?
thnks, great work!

**Comment by jonfinley** [comment]
I'm working on an update to the docker container. But natively I believe it should work. I do not currently have a device to test.

**Comment by rubenix** [comment]
ok, just tried like this:

docker create \
    --name=monitorr \
    --restart=always \
    -v /home/pi/monitorr/app:/app \
    -e TZ=Europe/Andorra \
    -e PGID=1000 \
    -e PUID=1000 \
    -p 98:80 \
    monitorr/monitorr

Once I start the container logs shows the following:
standard_init_linux.go:211: exec user process caused "exec format error"

**Comment by jonfinley** [comment]
Yeah we don’t have the arm docker built. I’ll generate this later today. 

Jonathan Finley
Owner
Infinite 8 Enterprises, LLC
infinite8.co
e: jonathan@infinite8.co
p: 703 504 8289

Sent from my iPhone

The content of this email is confidential and intended for the recipient specified in message only. It is strictly forbidden to share any part of this message with any third party, without a written consent of the sender. If you received this message by mistake, please reply to this message and follow with its deletion, so that we can ensure such a mistake does not occur in the future.

**Comment by rubenix** [comment]
lovely, will check and let you know! stay safe

**Comment by ajhorvath20** [comment]
That would be great. Thanks @jonfinley

**Comment by jonfinley** [comment]
Sorry, I got behind, will be getting to this shortly.

**Comment by ajhorvath20** [comment]
Thank you so much. Much appreciated

**Comment by jonfinley** [comment]
I have not forgotten this. Had a couple errors I needed to spin out. but so far so great. I should be pushing updates this week.

**Comment by rubenixnagios** [comment]
we have not forgotten this neither 😄 
let me know if I can help somehow...

**Comment by ajhorvath20** [comment]
Oh yeah same here. I was almost going to ask few days ago but then I thought its too early :)

**Comment by ajhorvath20** [comment]
Hi @jonfinley I hope you are okay :)

**Comment by rubenixnagios** [comment]
hope it too!!
